_data/sitetext.yml

```diff
@@ -14,6 +14,11 @@ en: &DEFAULT_EN
     title:
     text:
 
+  about:
+    title: "Clinical Scientific Computing"
+    image: assets/img/csc-logo-text.svg
+    slogan: "We build and deploy medical software for routine clinical care"
```

Comment:
I don't think there should be a slogan. If there has to be I would prefer to use the mission statement: developing people platforms policy for digital health.

Reply:
Just for the avoidance of doubt, the screenshot above clips the hero image. The top of the homepage looks like this:

![image](https://user-images.githubusercontent.com/15610752/160797371-dce942b8-f9a1-4bf2-b1ba-f2fc81569679.png)

Regarding the mission statement – I think this comes back to the point @lucy-funnell made last year when she showed the site to her friends. Are the 3 Ps instantly understandable? Personally prefer a statement a lay person can interpret. Be good to hear others' thoughts too.

Reply:
I think usig the tag line "developing people platforms policy for digital health" is ok as long as you can immeadiate see definions of these terms explained, so maybe removing the "what we do" section. 

I think the hero image may dominate too much real estate without provide information on what we do, mainly becuase its an abstract shape rather than a diagram or photo that relates to us (see https://sheffield3dlab.com/). Could it posiblly be greyscaled in the background with the CSC log in the forefront along with the tag line? 

The gap between the logo and the hero image could be increased as its scrunched up for me: 

![image](https://user-images.githubusercontent.com/79843755/160881653-0afaba62-b0d3-4f45-a9a2-4147f5856294.png)

Reply:
Taking inspiration from Innolitcs, howsabouts just the tag line front and centre?

![image](https://user-images.githubusercontent.com/79843755/160884336-e4c8c20e-8f7e-4281-a1b7-791eb6bfa1bd.png)

Reply:
I think I prefer it with the hero image but appreciate it's a bit 'loud' and maybe it could be toned down. I understand the apprehension about including the mission statement but I think having the 3 columns immediately underneath helps explain it in detail. Almost everything we write on the website is filled with jargon, even our name 'Clinical Scientific Computing' is ambiguous - I think we have to expect that lay audiences won't understand this and they shouldn't be our target

Reply:
Yeah that's true. Will implement 3Ps.

Reply:
The general sense I get from discussions about the hero image is that it's a nice graphic/piece of art, but it's isolated and a bit abstract, which I agree with. Think we need to throw down some different designs, bit like Anil has done, and get a feel for what we like.